src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+    private final int bound = 100;
```

Comment:
this is a constant, add `static` and name it accordingly 
https://checkstyle.sourceforge.io/reports/google-java-style-20170228.html#s5.2.4-constant-names
```suggestion
    private final int bound = 100;
```